README.md

```diff
@@ -99,18 +115,28 @@ A request for execution should contain the following request parameters:
 * `operationName` - [*Optional*]: The name of the Operation in the Document to execute.
 * `variables` - [*Optional*]: Values for any Variables defined by the Operation.
 
+Note: Be aware that `query` is a misleading name as it can contain a string describing the operation that may be either a query, a mutation or a subscription.
+
 Note: depending on the serialization format used, values of the aforementioned parameters can be
 encoded differently but their names and semantics must stay the same.
 
 Note: specifying `null` in JSON (or equivalent values in other formats) as values for optional request parameters is equivalent to not specifying them at all.
 
 ## GET
 
-For HTTP GET requests, the query parameters should be provided in the query component of the request URL in the form of
+For HTTP GET requests, the query parameters MUST be provided in the query component of the request URL in the form of
```

Comment:
I totally agree that this should be changed to MUST.  I cannot imagine what could be done with a GET request that contained no operation.